content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -364,6 +385,7 @@ Variable length text.
 | MySQL         | `varchar(191)`   |
 | MongoDB       | `String`         |
 | SQLite        | `TEXT`           |
+| CockroachDB   | `text`           |
```

Comment:
Looks like the cockroachdb type is actually `STRING`. and `TEXT` is an alias to it
https://www.cockroachlabs.com/docs/v21.2/string#aliases

Reply:
Thanks, I've added `STRING` but included the `TEXT` alias in the table below